libraries/lib-numeric-formats/NumericConverter.cpp

```diff
@@ -101,6 +101,19 @@ void NumericConverter::ControlsToValue()
                mInvalidValue;
 }
 
+bool NumericConverter::SetTypeAndFormatName (const NumericConverterType& type, const NumericFormatSymbol& formatName)
```

Comment:
Not really related to this PR, but I scanned for all occurences of `NumericConverter` and notice mentions of it in `ParsedNumericConverterFormatter.cpp` in comments, which seem no longer correct.

Reply:
Also the comment in `NumericConverter.h` refers to the wrong place to find an explanation of the mini language.

Reply:
And `NumericTextCtrl.h` duplicates that comment